lottery/templates/lottery/signup.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>Hello!</h1>
+
+    <p>Enter your data below to sign up for our ticket lottery. You can read more about it on
+    <a href="{% url 'blog:article' slug='2016-07-01-registration' %}">our blog</a>.</p>
+    <p>We ask for only one submission per person.</p>
```

Comment:
Suggested addition, as it's a common question:
`That means if multiple employees of your company want to come, they must each individually submit.`